known vm-import-controller issues (#685)
harvester/docs: versioned_docs/version-v1.3/advanced/addons/vmimport.md

This can take a while based on the size of the virtual machine, but users should see `VirtualMachineImages` created for each disk in the defined virtual machine.

> If the source VM is placed in a folder, then you can specify the folder name in the optional `folder` field.

Reviewer: ```suggestion
If the source virtual machine is placed in a folder, you can specify the folder name in the optional `folder` field.
```

:::note 
OpenStack allows users to have multiple instances with the same name. In such a scenario, users are advised to use the Instance ID. The reconciliation logic tries to perform a name-to-ID lookup when a name is used.
:::

#### Known issues

> * **Source VM name is not RFC1123 compliant:** vm-import-controller will create the Harvester VM object with the same name as the source VM name. Harvester VM names need to be meet the K8s Object naming [criteria](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names).
> In case the source VM does not meet these criteria the VM import may fail. Users may need to rename the source VM to allow import to be successful.

Reviewer: ```suggestion
* **Source virtual machine name is not RFC1123 compliant**: When creating a virtual machine object, the vm-import-controller add-on uses the name of the source virtual machine, which may not meet the Kubernetes object [naming criteria](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names). You may need to rename the source virtual machine to allow successful completion of the import.
```

> * **VM image name too long:** vm-import-controller, labels each imported disk with a label of format `vm-import-$VMname-$DiskName`. In certain scenarios this can exceed 63 characters. Users will notice the following error in the vm-import-controller logs:

Reviewer: ```suggestion
* **Virtual machine image name is too long**: The vm-import-controller add-on labels each imported disk using the format `vm-import-$VMname-$DiskName`. If a label exceeds 63 characters, you will see the following error message in the vm-import-controller logs:
```

Reviewer 2: Done

    ```shell
    harvester-vm-import-controller-5698cd57c4-zw9l5 time="2024-08-30T19:20:34Z" level=error msg="error syncing 'default/mike-mr-tumbleweed-test': handler virtualmachine-import-job-change: error creating vmi: VirtualMachineImage.harvesterhci.io \"image-z
    nqsp\" is invalid: metadata.labels: Invalid value: \"vm-import-mike-mr-tumbleweed-test-mike-mr-tumbleweed-test-default-disk-0.img\": must be no more than 63 characters, requeuing"      
    ```

>     Users may need to rename the source VM to allow import to be successful.

Reviewer: ```suggestion
    You may need to rename the source virtual machine to allow successful completion of the import.
```

Reviewer 2: Done
